target/linux/generic/hack-4.19/703-swconfig_b53_mdio_add_of_compatble.patch

```diff
@@ -0,0 +1,62 @@
+--- a/drivers/net/phy/b53/b53_mdio.c
```

Comment:
This patch here doen't patch a file from upstream, but rather the one tracked downstream here:

https://github.com/openwrt/openwrt/blob/master/target/linux/generic/files/drivers/net/phy/b53/b53_mdio.c

So it's more convenient to make the changes there instead of tracking this patch.

Reply:
No idea why I didn't do this in the first place. Fixed